Makefile

```diff
@@ -165,6 +165,16 @@ check-scripts:
     # Fail if any of these files have warnings
 	scripts/shellcheck.sh
 
+## check-version:           run check to ensure version in CHANGELOG.md matches references in files
+.PHONY: check-version
+check-version:
+    # Fail if syncing version would produce changes
+	scripts/version-sync.sh -c \
+	    -f README.md api-release \
+		-f preprocessing-pipeline-family.yaml release \
```

Comment:
why are the above 2 lines at a different indent level?

Reply:
Because it turns out most of the indenting in that file is done by tabs, and I didn't realize that and used spaces, and whatever you are using to view files indents tabs with more than 4 spaces while mine uses 4. I replaced the spaces with tabs to be consistent with the rest of the file.